src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -310,6 +426,75 @@ const FilterableWorkflowTable = ({
                           }
                         },
                       },
+                      {
+                        size: { basis: 325, grow: 0 },
+                        field: 'taskData',
+                        headerRenderer: () => 'Workflow Data',
+                        cellRenderer: ({ rowIndex }) => {
+                          const style = {
+                            display: 'grid',
+                            gridTemplateColumns: '1fr 1fr 1fr',
+                            gridColumnGap: '1em',
+                            gridRowGap: '0.3em',
+                          };
+                          if (paginatedPreviousRuns[rowIndex].engine_id) {
+                            return div({ style }, [
+                              h(
+                                Link,
+                                {
+                                  onClick: async () => {
+                                    showTaskDataModal('Inputs', null);
+                                    const workflow: WorkflowMetadata | undefined = await getWorkflow(rowIndex);
+                                    if (workflow !== undefined) {
+                                      // Perform this function to transform the keys to a more user friendly form, e.g. 'fetch_sra_to_bam.Fetch_SRA_to_BAM.SRA_ID' => 'SRA_ID'
```

Comment:
It would be nice for this comment to be in the docstring for `stripTaskPrefixFromKeys`. Then it would be accessible here by hovering over the function in an IDE.